addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -358,7 +358,8 @@ test("reply to message from inbox (message linked to document)", async () => {
         message_type: "comment",
         needaction: true,
         model: "res.partner",
-        res_id: partnerId,
+        res_id: serverState.partnerId,
+        author_id: partnerId,
```

Comment:
Nitpick: would move `author_id` alphabetically.
To bad we have to do it by hand, but with many props alphabetical order improves readability.